docs/General/Manuals/client-manual.rst

```diff
@@ -1480,6 +1480,171 @@ quick way to move around a large map in this manner.
+improvements that the city has produced and is supporting. Hovering your mouse over many of the items at the
+top of the :guilabel:`General` tab will show detailed calculation information on how the net value is
+produced.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-general.png
+  :align: center
+  :alt: Freeciv21 City Dialog General
+  :figclass: align-center
+
+  Figure 49: City Dialog - General
+
+
+We will discuss the :guilabel:`Citizens` tab in a bit. Moving to the bottom center, you will see a list of all
+the units that are present in the city as highlighted in Figure 50. There can be units present in a city that
+the city is not supporting, or from your allies. If you right-click on a unit, a pop up box shows some action
+commands that you can give to the unit. If you double-click on a unit, the :guilabel:`City Dialog` will close
+and the unit will be selected. See `Unit Controls`_ for more information on what to do with the unit.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-units.png
+  :align: center
+  :alt: Freeciv21 City Dialog Units
+  :figclass: align-center
+
+  Figure 50: City Dialog - Present Units
+
+
+Now let's come back and talk about the :guilabel:`Governor` tab on the left side segment and the
+:guilabel:`Citizens` tab on the right side segment.
+
+The :guilabel:`Governor` tab as highlighted in figure 51, gives information on the citizen management agent
+(CMA) for this city. For more information on how to use the CMA refer to :doc:`../../Playing/cma`.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-governor.png
+  :align: center
+  :alt: Freeciv21 City Dialog Govenor
+  :figclass: align-center
+
+  Figure 51: City Dialog - Govenor
+
+
+On the right side is the :guilabel:`Citizens` tab as highlighted in Figure 52. The :guilabel:`Citizens` tab
+shows you information about the happiness and the nationality of the citizens in the city. Happiness is
+broken down into segments: Cities (Empire Size), Luxuries (Luxury Goods), Buildings, Nationality, Units, and
+Wonders.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-citizens.png
+  :align: center
+  :alt: Freeciv21 City Dialog Citizens
+  :figclass: align-center
+
+  Figure 52: City Dialog - Citizens
+
+
+Overall happiness of the citizens in your cities depends heavily on all of these factors. Certain forms of
+government have varying degrees of empire size penalties and as your empire grows you will have to deal with
+the negative consequences of managing a large empire and the unhappiness it creates. The amount of Luxury
+Goods you are producing as part of your `Tax Rates View`_ will aid this problem. Certain city improvements can
+improve happiness as well as units in the city (martial law) along with wonders. If units are in the field
+in battle against your enemies, they can cause unhappiness. Hovering your mouse over these items will give you
+some information about it.
```

Comment:
Consider rephrasing to "Hovering your mouse over the appropriate row will show a pop-up with more detailed information."